randovania/games/cave_story/json_data/Labyrinth.json

```diff
@@ -1452,6 +1510,78 @@
+                            "type": "and",
+                            "data": {
+                                "comment": null,
+                                "items": []
+                            }
+                        },
+                        "Door to Camp (Lower)": {
+                            "type": "and",
+                            "data": {
+                                "comment": null,
+                                "items": []
+                            }
+                        },
+                        "Door to Camp (Upper)": {
+                            "type": "or",
+                            "data": {
+                                "comment": null,
+                                "items": [
+                                    {
+                                        "type": "and",
+                                        "data": {
+                                            "comment": null,
+                                            "items": [
+                                                {
+                                                    "type": "template",
+                                                    "data": "Can Break Blocks"
+                                                },
+                                                {
+                                                    "type": "resource",
+                                                    "data": {
+                                                        "type": "tricks",
+                                                        "name": "Dboost",
+                                                        "amount": 1,
+                                                        "negate": false
+                                                    }
+                                                }
+                                            ]
+                                        }
+                                    },
+                                    {
+                                        "type": "and",
+                                        "data": {
+                                            "comment": null,
+                                            "items": [
+                                                {
+                                                    "type": "template",
+                                                    "data": "Has Flight"
+                                                },
+                                                {
+                                                    "type": "template",
+                                                    "data": "Can Break Blocks"
+                                                }
+                                            ]
+                                        }
+                                    }
+                                ]
+                            }
```

Comment:
rather than being `(Can Break Blocks and Beginner Damage Boost) or (Can Break Blocks and Has Flight)`, this can be simplified to `Can Break Blocks and (Beginner Damage Boost or Has Flight). also, for the damage boost trick, make sure to add the minimum damage taken as a Damage resource - in this case 8HP.